tgui/packages/tgui/interfaces/PreferencesMenu/QuirksPage.tsx

```diff
@@ -129,6 +235,12 @@ const StatDisplay: StatelessComponent<{}> = (props) => {
 export const QuirksPage = (props, context) => {
   const { act, data } = useBackend<PreferencesMenuData>(context);
 
+  const randomToggleEnabled = false;
+
+  const randomBodyEnabled = false;
+  /* data.character_preferences.non_contextual.random_body !==
+      RandomSetting.Disabled || randomToggleEnabled;*/
```

Comment:
Commented code?

Reply:
never got back to this, there was a bug with random body that i never fixed, should be working in the next commit